tests/unit/test_client_gcs.py

```diff
@@ -1,17 +1,6 @@
 from datachain.client import Client
 
 
-def test_anon_url(mocker):
-    def sign(*args, **kwargs):
-        raise AttributeError(
-            "you need a private key to sign credentials."
-            "the credentials you are currently using"
-            " <class 'google.oauth2.credentials.Credentials'> just contains a token."
-            " see https://googleapis.dev/python/google-api-core/latest/auth.html"
-            "#setting-up-a-service-account for more details."
-        )
-
-    mocker.patch("gcsfs.GCSFileSystem.sign", side_effect=sign)
-
+def test_anon_url():
```

Comment:
As mentioned before, this will hit google cloud.

Reply:
🤔 Not sure if I understand how it can hit google cloud.
I also have tried it with debugger and can not find a way this to hit google cloud in current implementation.
If I miss something, please, help me to understand? 🙏

Reply:
Ah, I see — My apologies for the oversight. 😅